platform/osgi/platform-osgi-condpermadmin/src/main/java/org/codice/ddf/condpermadmin/PermissionActivator.java

```diff
@@ -66,7 +67,15 @@ public void start(BundleContext bundleContext) throws Exception {
     File policyDirFile = new File(policyDir);
     List<ParsedPolicy> parsedPolicies = new ArrayList<>();
     for (File file : Objects.requireNonNull(policyDirFile.listFiles())) {
-      parsedPolicies.add(new Parser(false).parse(file));
+      ParsedPolicy parse = null;
+      try {
+        parse = new Parser(false).parse(file);
+      } catch (Exception e) {
+        System.out.printf("%nUnable to parse policy file, %s. Please fix and try again.%n", file);
+        System.exit(1);
```

Comment:
❓ Should we not shutdown the system in a more official way like this? https://github.com/codice/ddf/blob/master/platform/security/secure-boot/src/main/java/org/codice/ddf/security/boot/SecureBoot.java#L117-L126

Reply:
I can give it a shot, but my guess is that we're actually too early in the process and are best served with a hard exit.

Reply:
Exit code 1? I'd say we pick a unique exit code (I imagine 1 is already overloaded with purposes)

Reply:
I wasn't particularly concerned with the exit code, other than it be non-zero. We certainly _can_ pick a specific value, but this is a fast failure condition and I'm not sure there's a lot of value in encoding the reason.